scripts/minikube.sh

```diff
@@ -67,10 +67,37 @@ function install_minikube() {
     curl -Lo minikube https://storage.googleapis.com/minikube/releases/"${MINIKUBE_VERSION}"/minikube-linux-"${MINIKUBE_ARCH}" && chmod +x minikube && mv minikube /usr/local/bin/
 }
 
+function detect_kubectl() {
+    if type kubectl >/dev/null 2>&1; then
+        command -v kubectl
+        return
+    fi
+    # default if kubectl is not available
+    echo '/usr/local/bin/kubectl'
+}
+
 function install_kubectl() {
+    local latest_version
+    latest_version=$(curl -L https://storage.googleapis.com/kubernetes-release/release/stable.txt 2> /dev/null)
+    if [[ -z "${latest_version}" ]]; then
+        # skip: getting latest version failed, continue with what we have
+        latest_version=${KUBE_VERSION}
```

Comment:
`KUBE_VERSION` should probably default to `latest` if you are adding the check for `release/stable.txt`

Reply:
@nixpanic, unfortunately, there is no tag like `latest` for kubectl like minikube.
But I will consider not updating the KUBE_VERSION variable.

Reply:
There is no `latest` tag, but you can have `KUBE_VERSION=latest` and then fetch the last stable version as you did. It would be helpful to be able to specify `latest` as default (so we do not run with old versions at one point).

Reply:
I hope you remember our yesterday's discussion about changing MINIKUBE_VERSION inside the script.

@nixpanic as part of install_kubectl() we will always fetch the latest kube version and store in a global variable (here latest_version, I'm changing it to KUBE_LATEST_VERSION in my next version of patches) which we will use through out this script.

And IMHO we should set KUBE_VERSION to something valid like tag `vx.y.z` so that we have something to fall back to, just in case if `curl -L https://storage.googleapis.com/kubernetes-release/release/stable.txt` fail for some practical reasons.

Thanks!

Reply:
Sorry, what I meant is, that I would love to see the _default_ `KUBE_VERSION` to be set to `latest` instead of `v1.14.10`.